modules/exploits/windows/http/sepm_auth_bypass_rce.rb

```diff
@@ -0,0 +1,113 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::FileDropper
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Symantec Endpoint Protection Manager Auth Bypass and RCE",
+      'Description'    => %q{
+      This module exploits three separate vulnerabilities in Symantec Endpoint Protection Manager
+      in order to achieve a remote shell on the box as NT AUTHORITY\SYSTEM
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'bperry', #metasploit module
+          'Markus Wulftange' #discovery
+        ],
+      'References'     =>
+        [
+          ['CVE', '2015-1486'],
+          ['CVE', '2015-1487'],
+          ['CVE', '2015-1489'],
+          ['URL', 'http://codewhitesec.blogspot.com/2015/07/symantec-endpoint-protection.html']
+        ],
+      'Payload'        => { 'BadChars' => "\x0d\x0a\x00" },
+      'Platform'       => 'win',
+      'Targets'        =>
+        [
+          [ 'Automatic', {
+            'Arch' => ARCH_X86,
```

Comment:
There are a couple targets it looks like I can add. One for a pure JSP payload for a command shell and a 64 bit one.

Sent from a phone

> On Aug 2, 2015, at 9:15 AM, Markus Wulftange <notifications@github.com> wrote:
> 
> In modules/exploits/windows/http/sepm_auth_bypass_rce.rb:
> 
> > +          'bperry', #metasploit module
> > +          'Markus Wulftange' #discovery
> > +        ],
> > +      'References'     =>
> > +        [
> > +          ['CVE', '2015-1486'],
> > +          ['CVE', '2015-1487'],
> > +          ['CVE', '2015-1489'],
> > +          ['URL', 'http://codewhitesec.blogspot.com/2015/07/symantec-endpoint-protection.html']
> > +        ],
> > +      'Payload'        => { 'BadChars' => "\x0d\x0a\x00" },
> > +      'Platform'       => 'win',
> > +      'Targets'        =>
> > +        [
> > +          [ 'Automatic', {
> > +            'Arch' => ARCH_X86,
> What about 64-bit Windows?
> 
> —
> Reply to this email directly or view it on GitHub.
>